frontend/src/components/parties/PartyProfile.js

```diff
@@ -221,7 +221,7 @@ export class PartyProfile extends Component {
                 {parties.phone_no} {parties.phone_ext ? `x${parties.phone_ext}` : ""}
               </p>
             </div>
-            <Address address={parties.address[0]} />
+            <Address address={parties.address[0] || {}} />
```

Comment:
Javascript question:  Does this mean: If parties.address[0] is truthy, use it else use '{}'

Reply:
Yep!